tests/e2e/operator/operator_test.go

```diff
@@ -99,33 +107,58 @@ var _ = Describe("Operator", Ordered, func() {
 		})
 
 		It("deploys all the CRDs", func() {
-			Eventually(kubectl.GetCRDs).
-				Should(ContainElements(crds), "Istio CRDs are not present; expected list to contain all elements")
+			Eventually(func(g Gomega) {
+				g.Expect(k8sClient.List(context.TODO(), customResourceDefinitionList)).To(Succeed(), "Error getting CRDs list from the cluster")
+
+				var crdNames []string
+				for _, crd := range customResourceDefinitionList.Items {
+					crdNames = append(crdNames, crd.ObjectMeta.Name)
+				}
+
+				g.Expect(crdNames).To(ContainElements(crds), "Istio CRDs are not present; expected list to contain all elements")
+			}).Should(Succeed(), "Unexpected error getting CRDs from the cluster")
 			Success("Istio CRDs are present")
 		})
 
 		It("updates the CRDs status to Established", func() {
 			for _, crd := range crds {
-				Eventually(kubectl.GetConditions).WithArguments(namespace, "crd", crd).Should(ContainElement(crdEstablished), "CRD is not Established")
+				Eventually(func(g Gomega) {
+					Expect(k8sClient.Get(context.TODO(), client.ObjectKey{Namespace: namespace, Name: crd}, customResourceDefintion)).
+						To(Succeed(), "Error getting CRD")
+
+					for _, condition := range customResourceDefintion.Status.Conditions {
+						if condition.Type == "Established" {
+							g.Expect(string(condition.Status)).To(Equal("True"), "CRD conditions was not the expected")
+						}
+					}
```

Comment:
We're also going to check for a specific condition in many different tests. We should have something that allows us to do it like this (or similar to this):
```suggestion
	Expect(getConditionStatus(crd.Status.Conditions, apiextensionsv1.Established)).To(Equal("True"))
```

(we should use the official constant instead of `"Established"`)

Or, instead of checking just the condition status, maybe it makes sense to check the entire condition? But in that case, you have to ignore the `lastTransitionTimestamp`.